prodtests/full-system-test/dpl-workflow.sh

```diff
@@ -431,6 +431,7 @@ has_detector FDD && ! has_detector_from_global_reader FDD && has_processing_step
 has_detector FV0 && ! has_detector_from_global_reader FV0 && has_processing_step FV0_RECO && add_W o2-fv0-reco-workflow "$DISABLE_DIGIT_ROOT_INPUT $DISABLE_ROOT_OUTPUT $DISABLE_MC"
 has_detector ZDC && ! has_detector_from_global_reader ZDC && has_processing_step ZDC_RECO && add_W o2-zdc-digits-reco "$DISABLE_DIGIT_ROOT_INPUT $DISABLE_ROOT_OUTPUT $DISABLE_MC"
 has_detector HMP && ! has_detector_from_global_reader HMP && has_processing_step HMP_RECO && add_W o2-hmpid-digits-to-clusters-workflow "$DISABLE_DIGIT_ROOT_INPUT $DISABLE_ROOT_OUTPUT --pipeline $(get_N HMP-Clusterization HMP REST 1 HMPCLUS)"
+has_detector HMP && ! has_detector_from_global_reader HMP && has_processing_step HMP_RECO && add_W o2-hmpid-matcher-workflow "--disable-root-input $DISABLE_ROOT_OUTPUT $DISABLE_MC"
```

Comment:
Your matcher produces `HMP/CONSTRAINED/0` (and `HMP/MC_CONSTRAINED/0`) outputs
https://github.com/AliceO2Group/AliceO2/blob/2b3de23a05d5c195513437fe8ad33ac9249a2e10/Detectors/GlobalTrackingWorkflow/src/HMPMatcherSpec.cxx#L163-L165, which can be consumed only by the writer https://github.com/AliceO2Group/AliceO2/blob/2b3de23a05d5c195513437fe8ad33ac9249a2e10/Detectors/HMPID/workflow/src/HMPMatchedWriterSpec.cxx#L50-L53. 
Then the reader produces the same output under the `HMP/MATCHES/0` (`HMP/MCLABELS/0`) specs which are consumed by the AODProducer via RecoContainer.
In the raw mode the data goes directly from the matcher to the AODProducer and obviously there is a mismatch between specs. You should change all specs above to  `HMP/MATCHES/0` (`HMP/MCLABELS/0`).

Reply:
> please use `$DISABLE_DIGIT_ROOT_INPUT` instead of explicit `--disable-root-input`

actually I put --disable-root-input there, since I though the matcher would never read ROOT input, also if other workflows would use ROOT input when running from digits?

But we can also homogeneously use $DISABLE_DIGIT_ROOT_INPUT, I have no strong opinion on this.

Reply:
I've asked to change it just for the homogeneity.